### PR TITLE
styling keycloak email template and reset password page

### PR DESCRIPTION
**Pull request by jingyi-gao-ttd**
Screenshot 2023-03-31 at 5 05 27 pm
Screenshot 2023-03-31 at 5 04 00 pm
Screenshot 2023-03-31 at 5 04 37 pm
image

### CODE REVIEW

File: keycloak/themes/uid2-theme/email/html/template.ftl [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<#macro emailLayout>
+<html>
+<body>
+  <div style="padding: 70px; text-align: center; font-family: 'Inter', sans-serif">
+    <div style="padding: 20px">
+      <a href="http://localhost:3000/"
+        ><img src="http://localhost:3000/uid2-logo.png" alt="UID2 logo" style="height: 30px"
```
Comment (Kshitij-Banerjee) [inline keycloak/themes/uid2-theme/email/html/template.ftl:7]: how would this get handled in prod ? would this need to be replaced with a cdn url ?

Comment (jingyi-gao-ttd) [reply keycloak/themes/uid2-theme/email/html/template.ftl:7]: Yes, this will be updated once we have a cdn for producation

Comment (lionell-pack-ttd) [reply keycloak/themes/uid2-theme/email/html/template.ftl:7]: @jingyi-gao-ttd can you add a ticket to Jira to make sure it gets done?

Comment (jingyi-gao-ttd) [reply keycloak/themes/uid2-theme/email/html/template.ftl:7]: @lionell-pack-ttd Here is the ticket https://atlassian.thetradedesk.com/jira/browse/UID2-985

**Review by lionell-pack-ttd — APPROVED** [review]

**Pull request merged**